src/Robot.Drivers/Sonar/HcSr04DistanceSensorDriver.cs

```diff
@@ -26,7 +26,7 @@ public HcSr04DistanceSensorDriver(HcSr04DistanceSensorDriverSettings settings, I
             Angle = settings.Angle;
             _sensor = new Hcsr04(settings.TriggerPin, settings.EchoPin, PinNumberingScheme.Logical);
             MeasuringCancellationToken = new CancellationTokenSource();
-            Task.Run(MeasurementCycle);
+            Task.Run(() => MeasurementCycle());
```

Comment:
Yeah, It's actually more performant than the method group call :)

Reply:
Oh sorry I changed this. I needed to because it was throwing a build error in my computer.
It was saying that there were two methods that matched the signature, or something like that.
We can revert it to how it was if we make it work everywhere

Reply:
No-no, this way is totally fine! I'm saying it even faster than the previous one 😁